pkg/retry/strategy.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package retry
+
+import (
+	"fmt"
+	"time"
+
+	tcontext "github.com/pingcap/dm/pkg/context"
+)
+
+// retryInterval represents enum of retry wait interval
+type retryInterval uint8
+
+const (
+	// LinearIncrease represents increase time wait retry policy, every retry should wait more time depends on increasing retry times
+	LinearIncrease retryInterval = iota + 1
+	// Stable represents fixed time wait retry policy, every retry should wait a fixed time
+	Stable
```

Comment:
should `Stable` be zero? it would be simpler

Reply:
Done